Makefile

```diff
@@ -118,12 +118,18 @@ docs_serve: build_docs
 docs_check_links:
 	${CONTAINER_RUNTIME} run -it -v $(CURDIR)/docs:/docs:Z --rm $(DOCS_BUILD_CONTAINER) docs_check_links
 
-.PHONY: clean_docs
+.PHONY: clean_docs clean_macos_package
 clean_docs:
 	rm -rf $(CURDIR)/docs/build
 
+clean_macos_package:
```

Comment:
Shouldn't we put `rm -fr packaging`

Reply:
`packaging/darwin` has some source files managed in git